core/src/main/java/org/elasticsearch/rest/action/explain/RestExplainAction.java

```diff
@@ -75,9 +76,9 @@ public void handleRequest(final RestRequest request, final RestChannel channel,
             String defaultOperator = request.param("default_operator");
             if (defaultOperator != null) {
                 if ("OR".equals(defaultOperator)) {
-                    queryStringBuilder.defaultOperator(QueryStringQueryBuilder.Operator.OR);
+                    queryStringBuilder.defaultOperator(Operator.OR);
                 } else if ("AND".equals(defaultOperator)) {
-                    queryStringBuilder.defaultOperator(QueryStringQueryBuilder.Operator.AND);
+                    queryStringBuilder.defaultOperator(Operator.AND);
```

Comment:
shall we use Operator.fromString here too? that is not case sensitive compared to the original code, maybe better?